plugin.video.jupiterbroadcasting/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon
+    id="plugin.video.jupiterbroadcasting"
+    name="Jupiter Broadcasting"
+    version="3.6.0"
+    provider-name="Jupiter Broadcasting">
+    <requires>
+        <import addon="xbmc.python" version="2.1.0" />
+        <import addon="script.module.beautifulsoup" version="3.2.1" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Jupiter Broadcasting video addon</summary>
```

Comment:
Can you adapt the language tags and language files for krypton. It should be en_GB, bg_BG